.github/workflows/cron.yml

```diff
@@ -0,0 +1,22 @@
+name: Weekly Cron
+
+on:
+  schedule:
+    - cron: "11 11 * * 4"
```

Comment:
oh and this was `* * * * 0` earlier? Is this correct?

Reply:
It was wrong cron syntax "* * * * 0" was every minute on sunday keep on pinging lucy xD